tests/models/TestApiMapper.php

```diff
@@ -26,16 +31,20 @@ public function buildLimit($resultsperpage, $start)
         ); // TODO: Change the autogenerated stub
     }
 
-    public function getPaginationLinks($list, $total = 0)
+    public function getPaginationLinks(array $list, $total = 0)
     {
         return parent::getPaginationLinks(
             $list,
             $total
         ); // TODO: Change the autogenerated stub
     }
 
-    public function inflect($string)
+    /**
+     * @param string $text
+     * @return string
+     */
+    public function inflect($text)
     {
-        return parent::inflect($string); // TODO: Change the autogenerated stub
+        return parent::inflect($text); // TODO: Change the autogenerated stub
```

Comment:
Consider it done. At least in another PR :) https://github.com/joindin/joindin-api/pull/426